docs/usage/configuration-options.md

```diff
@@ -1411,6 +1411,15 @@ This feature is independent of the `osvVulnerabilityAlerts` option.
 
 The source of these CVEs is [OSV.dev](https://osv.dev/).
 
+## dependencyDashboardReportAbandonment
```

Comment:
Do you think we need this? What would be the use of settings abandonmentThreshold in config if you don't display them in the dashboard?

Reply:
Well, at least it's available via logs